packages/composables/src/useCustomerOrders.ts

```diff
@@ -13,11 +13,22 @@ export type UseCustomerOrdersReturn = {
    *
    * In order to change a page with additional parameters please use `loadOrders` method.
    */
-  changeCurrentPage(pageNumber: number | string): Promise<void>;
+  changeCurrentPage(
+    pageNumber: number | string,
+    limit: number | string,
```

Comment:
is the limit necessary? It would be a breaking change

Reply:
Somehow, we need to pass the limit to the request.
We could also rely on the limit that is set by `total pages`, but that will indirectly couple this.